Sources/account/ALRegisterUserClientService.m

```diff
@@ -454,6 +454,20 @@ - (void)logoutWithCompletionHandler:(void(^)(ALAPIResponse *response, NSError *e
     }];
 }
 
+// Use this only when Logout APi Fails due to authorization
```

Comment:
Update the description to something like:
` This function changes the status to offline, unsubscribes from MQTT and clears the local data. Useful when the logout API is not working due to password change and you want to logout.`

Reply:
done